### ISSUE TITLE
1️⃣This emoji has not been accurately parse on parseTweet

### ISSUE THREAD

**Issue by eretica** [twitter/twitter-text#300]
1️⃣This emoji has not been accurately parse on parseTweet
version: 3.0.1

### Expected behavior

should be parsed as invalid

### Actual behavior

This snippet is officially invalid, but the result of parseTweet is determined to be valid.

### Steps to reproduce the behavior

```
1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣1️⃣
```

Do a parseTweet on this snippet

**Comment by leeaustinadams** [comment]
- What version of the library are you reproducing this on
- In what language are you testing (Java, Ruby, ObjectiveC, JavaScript)?
- If you are not compiling the library yourself, from where did you obtain the binary/package?
- Can you reply with the unicode that represents that emoji, it's hard to tell if GitHub's rendering of it is obscuring any facts.

In what way is the snippet officially invalid? It appears to be 140 emoji, which is valid.

Thanks

**Comment by eretica** [comment]
> これを再現するライブラリのバージョン

3.0.1

> In what language are you testing (Java, Ruby, ObjectiveC, JavaScript)?

Javascript

> If you are not compiling the library yourself, from where did you obtain the binary/package?

from package use by yarn

> Can you reply with the unicode that represents that emoji, it's hard to tell if GitHub's rendering of it is obscuring any facts.

I copied from the snippet, but it is invalid.

<img width="526" alt="スクリーンショット 2020-01-08 10 33 40" src="https://user-images.githubusercontent.com/16332352/71943015-7ad00d00-3202-11ea-9ec8-5268b661cb83.png">

**Comment by kaushlakers** [comment]
yes, this has been fixed internally and deployed in the app, but we haven't released to github yet.  We will publish a new version soon. Thanks!

**Comment by Ismael-Hery** [comment]
Hi. I have exactly this problem here. The fix will be greatly appreciated! Thank you very much.

**Comment by Ismael-Hery** [comment]
Hi @kaushlakers ,

Do you have any plan for a release in the next future?

Thank you very much for your help!

**Comment by t0m0120** [comment]
@kaushlakers 
Thanks for the bug fix!
When is the release likely?

**Comment by kaushlakers** [comment]
This should be fixed now in 3.1.0. Thanks for your patience here! 🙏🏽

**Comment by Ismael-Hery** [comment]
Thank you @kaushlakers!
